blockbase/sass/base/_header.scss

```diff
@@ -1,7 +1,10 @@
 .wp-site-blocks .site-header {
+	display: flex;
```

Comment:
`display:flex` shouldn't be necessary here since the layout type 'flex' is assigned in the template.

Reply:
:+1: Removing it in the next commit.